wp-content/themes/commons/library/scss/_wp-overrides.scss

```diff
@@ -84,7 +84,25 @@
 
   input[type=text] {
     line-height: 14px;
-    margin: 6px 0;
+    margin: 9px 0;
+  }
+}
+
+#searchform {
+  input[type=text] {
+    border: 1px solid lighten($commons-grey, 20%);
+    font-size: 14px;
+    padding: 5px 30px 5px 10px;
+    width: calc(100% - 45px);
```

Comment:
A fallback here would be nice (unless there already is somewhere else).

I would do a static amount for people with worse browsers, like:
```css
width: 255px;
width: calc(100% - 45px);
```